diktat-common/src/main/kotlin/org/cqfn/diktat/common/config/rules/RulesConfigReader.kt

```diff
@@ -116,6 +116,13 @@ data class CommonConfiguration(private val configuration: Map<String, String>?)
         configuration?.get("domainName")
     }
 
+    /**
+     * Get version of kotlin from configuration
+     */
+    val kotlinVersion: String? by lazy {
```

Comment:
I think we need some convinient wrapper to be able to quickly compare versions (like, in some rule use `if (kotlinVersion > KotlinVersion("1_3")` or something similar). Please check if there is something in kotlin compiler already.

Reply:
In `KotlinVersion` there is special method `isAtLeast`

Reply:
cool cool cool